scripts/constants.js

```diff
@@ -167,9 +167,28 @@ async function transform(constantsPath, templatePath, outputPath) {
   await writeFile(outputPath, mergedConstants.join('\n'));
 }
 
+async function substituteVersion(constantsPath) {
+    const packageJsonPath = './package.json';
+    const packageJsonSource = (await readFile(packageJsonPath)).toString('utf-8');
+
+    const json = JSON.parse(packageJsonSource);
+    const version = json.version;
+
+    const constantsInput = (await readFile(constantsPath)).toString('utf-8');
+    const constantsOutput = constantsInput.replace(
+        /(ReactNativeVoiceSDKVer\s*=\s*)(.*)/g,
```

Comment:
```suggestion
        /^(ReactNativeVoiceSDKVer=)(.*)$/g,
```

Reply:
why?

Reply:
After discussing in Slack, we've decided that we will leave the regex flexible in this PR, and then making validators down the line and make this regex more strict.